models/app.models.js

```diff
@@ -16,13 +16,23 @@ const fetchEndPoints = () => {
 
 const fetchArticleById = (id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = ${id}`)
+    .query(
+      `
+      SELECT articles.*, COUNT(comments.article_id) AS comment_count 
+      FROM articles 
+      JOIN comments ON comments.article_id = articles.article_id
+      WHERE articles.article_id = ${id}
```

Comment:
Big security risk here to the database 🚨 
What if someone puts in a path of _"/api/articles/;DROP+ARTICLES;"_ 
If you're putting values directly into the query they might be read by PSQL and are open to SQL injection, how can we protect against that ?
_hint: $1_